doc/modules/tree.rst

```diff
@@ -486,15 +486,25 @@ Regression criteria
 -------------------
 
 If the target is a continuous value, then for node :math:`m`,
-representing a region :math:`R_m` with :math:`N_m` observations, a common
-criterion to minimise is the Mean Squared Error
+representing a region :math:`R_m` with :math:`N_m` observations, common
+criteria to minimise are
+
+Mean Squared Error:
 
 .. math::
 
     c_m = \frac{1}{N_m} \sum_{i \in N_m} y_i
 
     H(X_m) = \frac{1}{N_m} \sum_{i \in N_m} (y_i - c_m)^2
 
+Mean Absolute Error:
+
+.. math::
+
+    c_m = \frac{1}{N_m} \sum_{i \in N_m} y_i
+
```

Comment:
I think it's fine to repeat the definition of `c_m`, it makes the definition of MAE. But instead of `c_m` I would actually use the following notation for that variable name:

```
\bar{y_m} = \frac{1}{N_m} \sum_{i \in N_m} y_i
```